packages/grid/x-data-grid-generator/src/renderer/renderEditStatus.tsx

```diff
@@ -89,10 +74,10 @@ function EditStatus(props: GridRenderEditCellParams) {
 
         return (
           <MenuItem key={option} value={option}>
-            <ListItemIcon className={classes.optionIcon}>
+            <ListItemIcon sx={{ minWidth: 40 }}>
```

Comment:
```suggestion
            <ListItemIcon sx={{ minWidth: 36 }}>
```

Reply:
depends if there is a reason to change the previous value

Reply:
I don't remember why I used 40px but I changed to 36px in f0ad99d.